packages/electron-commons/test/initializers/electron-node.spec.ts

```diff
@@ -1,60 +1,31 @@
 import fs from '@file-services/node';
-import { BaseHost, Communication } from '@wixc3/engine-core';
-import { initializeNodeEnvironment, ProcessExitDetails } from '@wixc3/engine-electron-commons';
-import { findFeatures } from '@wixc3/engine-scripts';
+import type { ProcessExitDetails } from '../../src';
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import type { IOType } from 'child_process';
-import { deferred } from 'promise-assist';
 import { platform } from 'os';
-import testFeature, { serverEnv } from '../test-project/test-feature.feature';
+import testFeature, { ErrorTypeConfig, serverEnv } from '@fixture/disconnecting-env/dist/disconnecting-env.feature';
+import { setupRunningNodeEnv } from '../../test-kit/setup-running-node-env';
 
 const { expect } = chai;
 chai.use(chaiAsPromised);
 
-const testProjectPath = fs.join(__dirname, '../test-project');
+const featurePath = fs.dirname(require.resolve('@fixture/disconnecting-env/package.json'));
 
-const setupRunningEnv = async ({
-    errorMode,
-    handleUncaught,
-    stdio = 'ignore',
-}: {
-    errorMode?: 'exception' | 'exit' | 'promiseReject' | 'out-of-memory' | 'no-error';
-    handleUncaught?: boolean;
+interface SetupRunningFeatureOptions {
+    config: Partial<ErrorTypeConfig>;
```

Comment:
Options object that contains a config object? Please no.

Reply:
`options` - the options of environment initialization
`config` - config of feature that is being initialized

Reply:
Then call it featureConfig. Isn't it weird that feature config has type `ErrorTypeConfig`?

Reply:
Also, `errorMode` in feature config? Are we taking about the same feature config?
https://github.com/wixplosives/engine/blob/master/packages/core/src/entities/config.ts#L9